src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/synthetic-monitoring-best-practices-guide.mdx

```diff
@@ -13,115 +13,130 @@ redirects:
-  </Collapser>
+    <CollapserGroup>
+    <Collapser
+        id="summary-ui"
+        title="Entity explorer"
+    >
+        To view a list of monitors::
 
-  <Collapser
-    id="overview-ui"
-    title="Monitors index page"
-  >
-    To view a list of monitors using the [Monitors index](/docs/synthetics/new-relic-synthetics/pages/synthetics-monitors-index-access-your-monitors) page:
+        Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities)**</DoNotTranslate> >  <DoNotTranslate>**Synthetic monitoring**</DoNotTranslate>.
 
-    Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Synthetic Monitoring**</DoNotTranslate>.
-  </Collapser>
-</CollapserGroup>
+        For more information, see [our docs about exploring your data](/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer/).
+    </Collapser>
 
-## 4. View individual monitor results [#view]
+    <Collapser
+        id="overview-ui"
+        title="Monitors index page"
+    >
+        To view a list of monitors using the [Monitors index](/docs/synthetics/new-relic-synthetics/pages/synthetics-monitors-index-access-your-monitors) page:
 
-You need to view the performance of your web apps as they're accessed from different parts of the globe. Your [Results](/docs/synthetics/new-relic-synthetics/pages/synthetics-results-access-individual-monitor-runs) page shows how everything from development to production affects user experience. You can sort what's listed to better identify problem areas or unusual results. Try filtering by location to compare monitor performance from different locations. To do this, you'll:
+        Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Synthetic Monitoring**</DoNotTranslate>.
+    </Collapser>
+    </CollapserGroup>
+    </Step>
 
-1. Go to <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Synthetic Monitoring**</DoNotTranslate>. 
-2. From the[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) >
-3. Click <DoNotTranslate>**Monitor**</DoNotTranslate>, then click <DoNotTranslate>**Results**</DoNotTranslate>. 
+    <Step>
+    ## View individual monitor results [#view]
 
-Results gives you an up-to-the-minute view of the slowest page loads for every monitored location.
+    You need to view the performance of your web apps as they're accessed from different parts of the globe. Your [Results](/docs/synthetics/new-relic-synthetics/pages/synthetics-results-access-individual-monitor-runs) page shows how everything from development to production affects user experience. You can sort what's listed to better identify problem areas or unusual results. Try filtering by location to compare monitor performance from different locations. To do this, you'll:
```

Comment:
Do they need to view them or do they want to view them or do they need to view them sometimes? "You need to" is a bit strong in my opinion, but this is just a small thing.

Reply:
Yes, for me it's the same. I changed it. Thanks!